src/librbd/cache/ObjectCacherObjectDispatch.cc

```diff
@@ -326,9 +326,9 @@ bool ObjectCacherObjectDispatch<I>::write(
   SnapContext snapc;
   if (io_context->write_snap_context()) {
     auto write_snap_context = *io_context->write_snap_context();
-    snapc = std::move(SnapContext(write_snap_context.first,
+    snapc = SnapContext(write_snap_context.first,
                                   {write_snap_context.second.begin(),
```

Comment:
Nit: can you update the indentation to match? Thanks.

Reply:
@dillaman done